src/lib/components/pages/anime_info/error.svelte

```diff
@@ -35,12 +35,12 @@
 
     <section
         transition:blur
-        class={cn(mapping.class, "relative grid h-full justify-end md:grid-flow-col")}
+        class={cn(mapping.class, "grid grid-cols-12 h-full")}
     >
         <error-context
             class:md:order-1={on_right}
             class:md:order-2={on_left}
-            class="row-span-5 flex flex-col items-center self-end leading-none md:mb-[13vw] md:items-start md:gap-[1vw] md:pl-[5vw]"
+            class="row-span-6 flex flex-col items-center self-end leading-none md:mb-[13vw] md:items-start md:gap-[1vw] md:pl-[5vw] col-span-12 md:col-span-6"
```

Comment:
`col-span-12` ? Explain? what's it doing?

Reply:
Heyo, I'm back, so uhh for small screen it gives full grid space
( doesnt work without this )